public/styles/shared.css

```diff
@@ -1,325 +1,411 @@
   font-size: 1.35rem;
   padding: 1rem;
-  position:relative;
+  position: absolute;
   top: 0;
-  font-family: "Linotte", sans-serif;
+  left: 0;
+  right: 0;
+  font-family: var(--font-brand);
   font-size: 1.5em;
   font-weight: bold;
   line-height: 1.4em;
-  z-index:999;
+  z-index: 999;
 }
 
-header.revealed{
-  visibility:visible;
-  position:sticky;
-  box-shadow:0px -3px 20px 1px rgba(0,0,0,.3);
-  background:white;
+.header__revealed {
+  --color-text: var(--color-midnight);
+  --color-link-text: var(--color-midnight);
+  background-color: var(--color-linen);
+  visibility: visible;
+  position: sticky;
+  box-shadow: 0px -3px 20px 1px rgba(0, 0, 0, 0.3);
+  background: white;
 }
 
-.header--contents {
+.header-nav {
   display: flex;
   flex-direction: row;
   justify-content: space-between;
-  margin: auto;
-  max-width:60rem;
+  max-width: 60rem;
   max-width: calc(100% - 60px);
   width: 100%;
+  padding: 0 1rem;
 }
 
-header .page-link a {
-  text-transform:uppercase;
-  text-decoration:none !important;
-  font-weight:600;
-  font-size:1.2rem;
-  color: #050422;
+.header-navlist {
+  list-style: none;
+  margin: 0;
+  padding: 0;
 }
 
-.logo {
-  font-size: inherit;
-  margin: 0;
+.header-navitem {
+  display: inline-block;
+}
+
+.header-navitem + .header-navitem {
+  margin-left: 2rem;
+}
+
+.header-navlink {
+  text-transform: uppercase;
+  text-decoration: none;
+  font-weight: 600;
+  font-size: 1.2rem;
+  display: inline-block;
+  color: var(--color-link-text);
   position: relative;
-  font-weight:700;
-  top:4px;
+  line-height: 1.5;
+  transition: all 0.2s ease-in-out;
 }
 
-.logo a{
-  text-decoration:none;
-  color: #050422;
+.header-navlink__home {
+  color: var(--color-link-text);
+  text-decoration: none;
 }
 
-header .page-link.blog{
-  margin-right:40px;
+.header-navlink__active:before {
+  content: " ";
+  background-color: var(--color-link-underline);
+  height: 1rem;
+  width: calc(100% + 25px);
+  left: -12px;
+  bottom: 0;
+  position: absolute;
+  z-index: -1;
+  border-radius: 20px;
 }
 
-header .page-link.faq{
-  margin-right:40px;
+.header-navlink:hover:not(.header-navlink__active) {
+  transform: scale(1.07);
 }
 
-.tag-tilt {
-  background: #FE3466;
-  color: white;
-  font-size: 1rem;
-  padding: .01rem 0.4rem;
+@media screen and (min-width: 768px) {
+  .header {
+    transform: none;
+  }
+}
+/**
+
+ .o88b.  .d88b.  d8b   db d888888b d88888b d8b   db d888888b 
+d8P  Y8 .8P  Y8. 888o  88 `~~88~~' 88'     888o  88 `~~88~~' 
+8P      88    88 88V8o 88    88    88ooooo 88V8o 88    88    
+8b      88    88 88 V8o88    88    88~~~~~ 88 V8o88    88    
+Y8b  d8 `8b  d8' 88  V888    88    88.     88  V888    88    
+ `Y88P'  `Y88P'  VP   V8P    YP    Y88888P VP   V8P    YP 
+ */
+
+.container--md {
+  max-width: calc(640px + 4rem);
+  padding: 4rem 2rem 0;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.section--content,
+.section--callout {
+  position: relative;
+  margin: auto;
+  padding: 5rem 0;
```

Comment:
On the FAQ page, http://localhost:8080/faq.html:

The text "Let us know if you have other questions here." has some extra padding on top and bottom.